code/modules/mob/mob_movement.dm

```diff
@@ -83,6 +83,15 @@
 	if(SEND_SIGNAL(mob, COMSIG_MOB_CLIENT_PRE_MOVE, args) & COMSIG_MOB_CLIENT_BLOCK_PRE_MOVE)
 		return FALSE
 
+	if (istype(L.get_item_by_slot(SLOT_HEAD), /obj/item/clothing/head/helmet/f13/power_armor) || istype(L.get_item_by_slot(SLOT_HEAD), /obj/item/clothing/head/helmet/f13/heavy/salvaged_pa))
```

Comment:
hahahahahahaha you straight up put it in fucking `client/Move` instead of using signals hahahahahahahahahaha

Reply:
this is not helpful